tests/test_barely_json.py

```diff
@@ -246,7 +244,11 @@ def test_nested(self):
         )
 
     def test_custom_resolver(self):
-        resolver = lambda value: 1 if value == 'one' else value
+        def resolver(value_):
```

Comment:
A matter of taste, I guess. I can live with the longer form, but note that the comment regarding `value_` also applies here.